### DISCUSSION TITLE
Smooth lighting.

### DISCUSSION THREAD

**Discussion by JustCauseWhyNot** [AnOpenSauceDev/Methane-mod#37] [Q&A] [answered]
Does smooth lighitng have any effect when methane's inststalled, and should it just be disabled?

**Comment by AnOpenSauceDev** [comment, marked as answer]
> Does smooth lighitng have any effect when methane's inststalled, and should it just be disabled?

Smooth lighting is basically ambient occlusion (AO), and doesn't really have much to do with lighting.
Yes, you should get that "smoothed" look on block edges.

**Reply by JustCauseWhyNot** [reply]
You're right. Thanks.
